docs/faq.md

```diff
@@ -33,5 +33,9 @@ Please check whether you are doing blocking jobs (e.g. sleep, IO) in your task.
 
 Generally, we recommend use the default `share-thread-pool-dispatcher` which has better performance and only turn to the `single-thread-dispatcher` when you have to.
 
+##### Why can't I open Dashboard even if the Services process has been launched successfully ?
+
+By default, our Services process binds to a local **IPv6 port**. It's possible that another process on your system has already taken up the same **IPv4 port**. You may check by `lsof -i -P | grep -i "Listen"` if your system is Unix/Linux. 
```

Comment:
You mean 
When:
1. When another process take the ipv6 port 8090.
2. UI successfully bind to ipv4 port 8090.

the user cannot open dashboard of the ipv4:8090?